### ISSUE TITLE
CELA P1_A11y_code.visualstudio.com_WCAG2.1_Homepage_AI4D: Color contrast ratio for the "Stable Build" Text present in Download for windows button is less than 4.5:1

### ISSUE THREAD

**Issue by khansahab15** [microsoft/vscode-docs#3427]
“[Check out Accessibility Insights!](https://accessibilityinsights.io/) - Identify accessibility bugs before check-in and make bug fixing faster and easier.”

GitHubTags: #A11ySev2; #A11yTCS; #A11yWCAG2.1; #Benchmark; #CELA P1; #ChromiumEdge; #ColorContrast; #DesktopWeb; #MAS1.4.11; #Win10; #Visual Studio Code; #AI4D; #CELA P1_A11y_code.visualstudio.com_WCAG2.1

**Environment Details:**
Application Name: code.visualstudio.com
#URL:https://code.visualstudio.com/
Browser version: Chromium Edge 80.0.361.54
Windows Version: Windows10

**Repro Steps:**
1.	Launch  website application using URL: https://code.visualstudio.com/
2.	"Home" screen gets open.
3.	Open Accessibility Insight for desktop and check the contrast ratio of "Stable Build" Text present in Download for windows button.

**Actual:**
Color contrast ratio for the "Stable Build" Text present in Download for windows button is less than 4.5:1.

**Same issue is present with all the buttons present throughout the page.**

**Expected:**
Color contrast ratio for the "Stable Build" Text present in Download for windows button should be >= 4.5:1.

**User Impact:**
For the Low vision user it will be difficult to identify the content of the Button.

**Attachment:**
[#3427.zip](https://github.com/microsoft/vscode-docs/files/4260337/3427.zip)

**Comment by misolori** [comment]
This should be fixed now:

![image](https://user-images.githubusercontent.com/35271042/78283688-017b4a00-74d3-11ea-900e-86df7e757082.png)

**Comment by Yashi1201** [comment]
Issue is fixed in env https://code.visualstudio.com/
Please refer attachement. Hence, closing the bug.
![3427_Issue fixed](https://user-images.githubusercontent.com/93374043/216566967-c3d29255-7605-44d0-9772-26ff9c6a0f6e.png)

**GithubTags**:#Closed;

**Comment by 12-shweta** [comment]
CELA_code.visualstudio.com_WCAG2.1_FEB2020;

**Comment by InduPriya1805** [comment]
GithubTags:#CELA_code.visualstudio.com_WCAG2.1_FEB2020;
